src/url-parameters/url-parameters.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { UrlParametersComponent } from './url-parameters.component';
+
+describe('UrlParametersComponent', () => {
+  let component: UrlParametersComponent;
+  let fixture: ComponentFixture<UrlParametersComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ UrlParametersComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(UrlParametersComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
@Miffael why you didn't check the calling `parseString` `dispachToStore` `combineUrlPerPage` of `urlParametersService`. and why you didn't stub `urlParametersService`?